demo/scripts/column-chart.js

```diff
@@ -164,10 +164,14 @@ module.exports = {
     init: function(){
         var demos = ['year','yearWithNegative','years','yearsWithNegative','decade', 'month', 'multiple', 'time'];
         //var demos = ['multiple'];
-        demos.forEach(function(timeFrame){
-            d3.select('#views').append('div').attr({
+        demos.forEach(function(timeFrame, i){
+            var textContent = '';
+            if (i===7){
+                textContent = 'NOTE: This chart highlights how columns should rarely be used for time-data. This example should check that charts of this form render but not that they should look good.'
```

Comment:
changed wording slightly to make it clearer that all we're looking for is that they render.